template/android/gradle.properties

```diff
@@ -9,13 +9,16 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
```

Comment:
same here

Reply:
Actually, the update to this file has been missing for a very long time. The **default** Android project templates as generated by Android Studio include the setting of the jvmargs to `-Xmx1536m`. Running with lower heap space might degrade performance or fail the build. You can check by just generating any new project in Android Studio and open the `gradle.properties` file.

Reply:
Those default is fit for project like react native, it's likely you can't run on machine memory like than 8GB.